addons/crm/models/res_partner.py

```diff
@@ -61,7 +61,13 @@ def action_view_opportunity(self):
         This function returns an action that displays the opportunities from partner.
         '''
         action = self.env['ir.actions.act_window']._for_xml_id('crm.crm_lead_opportunities')
-        action['context'] = {}
+        action['context'] = {
+            'search_default_filter_won': 1,
+            'search_default_filter_ongoing': 1,
+            'search_default_filter_lost': 1
```

Comment:
Should be a note in the commit message about that, mentioning we activate those to match the stat button statistics.